path-manager/app/services/PathStore.scala

```diff
@@ -133,6 +198,21 @@ object PathStore extends Logging {
     }
   }
 
+  def deleteAlias(path: String) = {
```

Comment:
reading the functionality, does this not delete all aliases associated with the supplied path? 

If so could it be...
```suggestion
  def deleteAliasesAssociatedWithPath(path: String) = {
```

If not, should `pathItems` in the function body not be something like `maybePathItem` (with an exception or Either to handle the 'else scenario' of the path not being an alias or if no records match that path).

Reply:
No, it just deletes that one alias. The `map` is to handle an `Option`. This isn't clear, so I tidied it a bit. We might just want to delete this method after our chat today though.